Sentinel-dVPN-node/deploy.yml

```diff
@@ -1,37 +1,35 @@
 ---
 version: "2.0"
 endpoints:
-  your_endpoint_name:  # # WARRING! SPECIFY UNIQ ENDPOINT NAME! ONLY LOWER CASE SYMBOLS!
+ <uniq_name_endpoint>: # unique name, for example "dvpn_dimokus:", and enter this name in line 23 and 28.
```

Comment:
let's just keep a working (real) endpoint name here so people can deploy it with the less modifications.

You can keep the comment that the endpoint name must be unique until https://github.com/akash-network/support/issues/108 is resolved